spec/javascript/e2e/SupplementalApplicationPage/preferences.e2e.js

```diff
@@ -7,55 +7,142 @@ import {
+    const typeOfProofToSetValue = await page.$eval(unselectedTypeOfProofSelector, e => e.value)
+    await page.select(typeOfProofSelector, typeOfProofToSetValue)
+
+    // Update the preference status
+    const prefStatusSelector = `${liveWorkExpandedPanelSelector} .preference-status-select`
+    const unselectedPrefStatusSelector = getUnselectedOptSelector(prefStatusSelector)
+    const prefStatusToSetName = await page.$eval(unselectedPrefStatusSelector, e => e.textContent)
+    const prefStatusToSetValue = await page.$eval(unselectedPrefStatusSelector, e => e.value)
+    await page.select(prefStatusSelector, prefStatusToSetValue)
+
+    // Click the save button in the preference panel
+    await page.click(`${liveWorkExpandedPanelSelector} .save-panel-btn`)
+
+    // Wait for the API preference update call to complete
+    await page.waitForResponse(request => request.url().includes('http://localhost:3000/api/v1/preferences/'))
+
+    // Validate that the panel has collapsed
+    const isCollapsed = await page.$eval(liveWorkExpandedPanelSelector, e => e.attributes['aria-hidden'].value)
+    expect(isCollapsed).toBe('true')
+
+    // Validate that values were updated in the table.
+    const liveWorkRowValues = await page.$$eval(`${liveWorkRowSelector} td`, tds => tds.map((td) => td.textContent))
+
+    expect(liveWorkRowValues[1]).toContain(prefToSetName.split(' ')[0]) // Check if both contain 'Live' or 'Work'. TODO: Find a less hacky way to do this
+    expect(liveWorkRowValues[4]).toBe(typeOfProofToSetName)
+    expect(liveWorkRowValues[5]).toBe(prefStatusToSetName)
+
+    // Reload the page
+    await sharedSteps.goto(page, `/applications/${LEASE_UP_LISTING_APPLICATION_ID}/supplementals`)
+
+    // Open the same preference edit panel as before
+    await page.waitForSelector(`${liveWorkRowSelector}`)
+    await page.click(liveWorkEditSelector)
 
-    // Find an unselected option value in the panel's Status select field
-    const statusToSetName = await page.$eval(unselectedOptSelector, e => e.textContent)
-    const statusToSetValue = await page.$eval(unselectedOptSelector, e => e.value)
+    // Check that the value entered for individual preference, type of proof, and status were successfully saved
+    // and now appears in the preference panel
+    const currentIndividualPref = await page.$eval(getSelectedOptSelector(individualPreferenceSelector), e => e.textContent)
+    expect(currentIndividualPref).toBe(prefToSetName)
+
+    const currentTypeOfProof = await page.$eval(getSelectedOptSelector(typeOfProofSelector), e => e.textContent)
+    expect(currentTypeOfProof).toBe(typeOfProofToSetName)
+
+    const currentStatus = await page.$eval(getSelectedOptSelector(prefStatusSelector), e => e.textContent)
+    expect(currentStatus).toBe(prefStatusToSetName)
+
+    await browser.close()
+  }, DEFAULT_E2E_TIME_OUT)
+
+  test('should allow updates to assisted housing preference', async () => {
```

Comment:
@akegan looks like the comments in this test need to be updated from referencing live/work to reference the assisted housing pref